voltdb/datadog_checks/voltdb/check.py

```diff
@@ -0,0 +1,148 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from typing import Any, List, Optional, cast
+
+import pkg_resources
+import requests
+from six import raise_from
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.base.utils.db import Query, QueryManager
+
+from . import queries
+from .client import Client
+from .config import Config
+from .types import Instance
+
+BASE_PARSED_VERSION = pkg_resources.get_distribution('datadog-checks-base').parsed_version
+
+
+class VoltDBCheck(AgentCheck):
+    __NAMESPACE__ = 'voltdb'
+
+    def __init__(self, name, init_config, instances):
+        # type: (str, dict, list) -> None
+        super(VoltDBCheck, self).__init__(name, init_config, instances)
+
+        self._config = Config(cast(Instance, self.instance), debug=self.log.debug)
+        self.register_secret(self._config.password)
+        self._client = Client(
+            url=self._config.url,
+            http_get=self.http.get,
+            username=self._config.username,
+            password=self._config.password,
+            password_hashed=self._config.password_hashed,
+        )
+
+        manager_queries = [
+            queries.CPUMetrics,
+            queries.MemoryMetrics,
+            queries.SnapshotStatusMetrics,
+            queries.CommandLogMetrics,
+            queries.ProcedureMetrics,
+            queries.LatencyMetrics,
+            # queries.StatementMetrics,  # TODO: reconsider later, this is causing unsustainably too many data points
```

Comment:
Do we want to keep this here, may be perceived as an option and added by a user?

Reply:
Ah, of course, I need to unwrap those quick fixes. :-) Let's just drop these metrics altogether for now — I'll remove the comments.